view/frontend/templates/js/custom_sso_buttons.phtml

```diff
@@ -0,0 +1,100 @@
+    return;
+}
+
+if (!$block->isBoltSSOEnabled()) {
+    return;
+}
+
+$customSSOSelectors = $block->getCustomSSOSelectors();
+?>
+<script>
+    var boltSSOCustomClass = 'bolt-sso-custom';
+    window.whenDefined(window, 'BoltAccount', function () {
+        var selectors = <?php echo json_encode($customSSOSelectors); ?>;
+
+        /**
+         * Converts all present elements matching {@see loginSelectors} and {@see logoutSelectors}
+         * to custom Bolt SSO buttons
+         */
+        function initializeBoltSSOCustomLinks () {
+            var selectorConfig, link, linkIndex, links, selector;
+            for (selector in selectors) {
+                links = document.querySelectorAll(selector);
+                for (linkIndex = 0; linkIndex < links.length; linkIndex++) {
+                    link = links[linkIndex];
+                    selectorConfig = selectors[selector];
+                    if (selectorConfig.hasOwnProperty('logout') && selectorConfig.logout) {
+                        link.setAttribute('data-logged-in', 'true');
+                    }
+                    if (selectorConfig.hasOwnProperty('redirect') && selectorConfig.redirect) {
+                        link.setAttribute('data-destination-on-success', selectorConfig.redirect);
+                    }
+
+                    link.removeAttribute('href');
+                    link.removeAttribute('data-post');
+
+                    // add {@see boltSSOCustomClass} only once
+                    link.setAttribute(
+                        'class',
+                        (link.getAttribute('class') ?? '').replace(boltSSOCustomClass, '') + ' ' + boltSSOCustomClass
+                    );
+                    link.setAttribute('style', 'cursor:pointer;');
+                }
+            }
+            BoltAccount.injectButtons();
+        }
+
+        initializeBoltSSOCustomLinks();
+        require(['domReady!'], initializeBoltSSOCustomLinks);
+
+        require(['jquery'], function ($) {
+
+            /**
+             * Handles buttons matching configured selectors that weren't converted to Custom Bolt SSO buttons
+             */
+            $(document).on('click', Object.keys(selectors).join(','), function (e) {
+                if ($(e.currentTarget).hasClass(boltSSOCustomClass)) {
+                    return;
+                }
+                var redirect;
+                for (var selector in selectors) {
+                    if (e.currentTarget.matches(selector)) {
+                        var selectorConfig = selectors[selector];
+                        if (selectorConfig.hasOwnProperty('redirect') && selectorConfig.redirect) {
+                            redirect = selectorConfig.redirect;
+                        }
+                        if (selectorConfig.hasOwnProperty('logout') && selectorConfig.logout) {
+                            BoltAccount.logoutButtonClicked(redirect); //@todo verify that this is API
```

Comment:
Is this todo expected?

Reply:
@vitaliyreznikov It's there as a reminder to sync with the SSO / account team that we are using this calls
`BoltAccount.logoutButtonClicked, BoltAccount.loginButtonClicked`, the interface should not be changed.

Reply:
@mikeshutov-bolt Could you please review this PR? Looks like we need to check methods and remove todo record after this.